examples/cms-sitecore/components/hero-post.js

```diff
@@ -0,0 +1,39 @@
+import Avatar from '../components/avatar'
+import CoverImage from '../components/cover-image'
+import Date from '../components/date'
+import Link from 'next/link'
+
+export default function HeroPost({
+  title,
+  coverImage,
+  date,
+  excerpt,
+  author,
+  id,
+}) {
+  return (
+    <section>
+      <div className="mb-8 md:mb-16">
+        <CoverImage title={title} url={coverImage.url} id={id}  />
+      </div>
+      <div className="md:grid md:grid-cols-2 md:col-gap-16 lg:col-gap-8 mb-20 md:mb-28">
+        <div>
+          <h3 className="mb-4 text-4xl lg:text-6xl leading-tight">
+            <Link as={`/posts/${id}`} href="/posts/[id]">
```

Comment:
Any instances of the `as` prop can be removed - it's no longer needed! For example, this would just be `href={`/posts/${id}`}` now.